client/state/sites/selectors.js

```diff
@@ -43,7 +43,7 @@ import { isSiteUpgradeable } from 'state/selectors';
  * @return {?Object}        Site object
  */
 export const getRawSite = ( state, siteId ) => {
-	return state.sites.items[ siteId ] || null;
+	return get( state, [ 'sites', 'items', siteId ], null );
```

Comment:
Should we be using the `getSitesItems` selector and assuming it will return an object?

```js
return getSitesItems( state )[ siteId ] || null;
```

Reply:
> Should we be using the getSitesItems selector and assuming it will return an object?
>```javascript
>return getSitesItems( state )[ siteId ] || null;
>```

Empty object is truthy so this will never return `null`?

Reply:
Hi @aduth, sorry the delay I missed this comment. The change was applied.
@vindl if the siteId is not found getSitesItems( state )[ siteId ] will return undefined so we use getSitesItems( state )[ siteId ] || null to make sure we return null in that case as before instead of undefined.